client/homebrew/pages/vaultPage/vaultPage.jsx

```diff
@@ -34,7 +37,10 @@ const VaultPage = (props)=>{
 
 	useEffect(()=>{
 		disableSubmitIfFormInvalid();
-		loadPage(pageState, true);
+		const initialSort = props.query.sort || 'title';
+		const initialDir = props.query.dir || 'asc';
+		const initialSorting = `${initialSort}/${initialDir}`;
+		loadPage(pageState, true, initialSorting);
```

Comment:
Any special reason `initialSorting` needs to be a string that you join and split up later? Seems it would be more straightforward to just pass the sort category and direction as separate parameters? Or if the must be combined, as an object?

Reply:
there is no need for separate parameters, it is overall less words, i think. don't really care

Reply:
But it adds an unnecessary step of combining/splitting the string, which is a very awkward way of doing things. If there is no special reason let's just keep them separate.

Reply:
solved